### ISSUE TITLE
Passing headless false fails with an error

### ISSUE THREAD

**Issue by leweafan** [elastic/synthetics#782]
Passing `headless: false` I get an error "Looks like you launched a headed browser without having a XServer running".

According to https://github.com/elastic/synthetics/issues/740 this issue has been fixed.

Tested with Elastic stack 8.7.1

**Comment by leweafan** [comment]
@vigneshshanmugam can you please help. You was the one the one who fixed it before.

**Comment by vigneshshanmugam** [comment]
@leweafan Thanks for reporting the issue, Could you tell us where you are running the tests? We dont support the headfull mode on our global synthetics managed infrastructure yet.

**Comment by leweafan** [comment]
I run it locally on self hosted stack.

**Comment by vigneshshanmugam** [comment]
@leweafan Thanks, To enable the tests to run on Headed mode, you would need to manually invoke the `xvfb-run` command before starting the test itself. 

Check the docs here from Playwright - https://playwright.dev/docs/ci#running-headed

**Comment by leweafan** [comment]
@vigneshshanmugam I use elastic-agent-complete docker container. How I can invoke the xvfb-run using elastic-agent-complete?

**Comment by paulb-elastic** [comment]
Hi @leweafan, can you share a little more about what you're trying to achieve here?  With the intention being for these monitors to run regularly _unattended_, it would be useful to understand what your use case is for needing it to be headful?

**Comment by leweafan** [comment]
Hello @paulb-elastic! 
We have got a problem with site rendering and our investigation shows that the problem with user_agent. If we use HeadlessChrome it fails that is why I decided to try `headless: false` option. Cause we use elastic-agent-complete docker container I get an error "Looks like you launched a headed browser without having a XServer running".
Seems like option `headless: false` is not compatible with elastic-agent-complete. 
For now I can solve the issue using useragent option but still interested in `headless: false` option.
